app/(tabs)/addnew/index.tsx

```
@@ -1,328 +1,302 @@
-  const {id} = useLocalSearchParams();
-  const expense = useAppSelector(selectExpense(+id)) || initState();
-  const categories = useAppSelector(selectCategories);
 
-  // State for edit mode and editing fields
-  const [isEditMode, setIsEditMode] = useState(false);
-  const [editedExpense, setEditedExpense] = useState(initState(expense));
+  const focusRef = useRef<HTMLInputElement>(null);
+  const dirty = useRef({});
 
-  const isValid = validateInput(editedExpense);
+  const isPasRecord = isNaN(+id) ? false : true;
```

Biome (1.9.4)

[error] Unnecessary use of boolean literals in conditional expression.

Simplify your code by directly assigning the result without using a ternary operator.
If your goal is negation, you may use the logical NOT (!) or double NOT (!!) operator for clearer and concise code.
 Check for more details about NOT operator.
Unsafe fix: Remove the conditional expression with


(lint/complexity/noUselessTernary)